README.txt

```diff
@@ -24,7 +24,8 @@ so you may well want to use them. For example,
 $ java -classpath xom-samples.jar:xom-1.3.9.jar:lib/xml-apis.jar:lib/xercesImpl.jar nu.xom.samples.PrettyPrinter filename.xml
 
 junit.jar is only used for testing, and is
-not needed for normal operation of XOM.
+not needed for normal operation of XOM. XOM uses JUnit 4.13.2 with
```

Comment:
delete sentence " XOM uses JUnit 4.13.2 with
legacy runner support, so tests written for JUnit 3 continue to work."